.vscodeignore

```diff
@@ -1,23 +1,24 @@
-.vscode
-.vscode-test
+.github/
+.husky/
+.vscode/
+.vscode-test/
 out/
+scripts/
 src/
 test-fixtures/
-tsconfig.json
-webpack.config.js
-.prettierignore
-azure-pipelines.yml
-scripts/
-samples/
+.gitattributes
 .gitignore
-.github/
+.nvmrc
+.prettierignore
+.prettierrc
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+CONTRIBUTORS.md
 tsconfig.json
+webpack.config.js
 yarn.lock
-CONTRIBUTORS.md
-CONTRIBUTING.md
-CODE_OF_CONDUCT.md
 
-node_modules
+node_modules/
```

Comment:
I think not needed `/` after dir

Reply:
It is not needed but other directories had it so I added it for consistency and additionally it makes it clear it is a directory and not a file.